scripts/migrations/tournaments/main.go

```diff
@@ -500,17 +454,91 @@ func main() {
 	}
 	log.Info().Interface("ids", ids).Msg("listed-game-ids")
 
-	for _, tid := range ids {
-		_, err := oldDatabaseObjectToEntity(ctx, tournamentStore, tid)
-		if err != nil {
-			panic(err)
-		}
+	tournamentStore.db.Transaction(func(tx *gorm.DB) error {
```

Comment:
`tournamentStore.db` is a private variable -- how's this compile?

Reply:
I've redefined NewDBStore in this script so I could have access to that field. See above.